### ISSUE TITLE
[RFC] Proposal to extend TAC call from 60 to 90mins every 2 weeks

### ISSUE THREAD

**Issue by SecurityCRob** [ossf/tac#391]
**Proposal** -Extend the regular TAC calls from 60min to 90min every two weeks.

**Background** - Historically the TAC has been rushed during the regular calls to address all open/desired topics and discussions have been limited in duration due to high volume of petitions.  Adding an additional 30min buffer onto the call will allow more conversation/debate/discussion to ensure all perspectives and data is accounted for in the TAC's decisions and processes.

**Comment by steiza** [comment]
I'm in favor of this. On the one hand, meetings will probably expand to fill the time allotted, but we really do have quite a bit to cover, and there have been good discussions when we have time for them. I think we should continue to make and adhere to a scheduled agenda, but having 90 minutes instead of 60 will allow more of that discussion time to happen.

**Comment by lehors** [comment]
I would prefer we do more work offline but I'm ok with this proposal.

**Comment by mlieberman85** [comment]
I agree with @lehors regarding more work offline. If possible I would rather have something like a weekly meeting instead of one 90 minute meeting every two weeks. If we did move to 90 minutes, I would need to either find a better time for the meeting as I have something right after the TAC meeting, or I would need to figure out how to move that other meeting.

**Comment by marcelamelara** [comment]
My preference would be to keep the current 60 min bi-weekly slot and encourage/facilitate more offline work. If the majority of the TAC votes for switching to 90 min, I would want to extend to 11:30 ET, rather than start 30 min earlier.

@mlieberman85 Given that our meeting following the TAC meeting is usually only attend by the two of  us and one other person, it might not be too challenging to find a different time slot for it?

**Comment by bobcallaway** [comment]
My first preference is to keep the current time slot and duration (with preference for work offline), but would prefer moving to a 90 min session instead of going to weekly.

**Comment by SecurityCRob** [comment]
TAC agreed to not move forward with extending the call at this time and will attempt to do more work asynchronously before meetings to optimize time.
